### DISCUSSION TITLE
Muliple Processes and Multiple Outputs

### DISCUSSION THREAD

**Discussion by fakharak** [swoole/swoole-src#4246] [Q&A] [answered]
I am running TCP Server on port 9501 and as SWOOLE_PROCESS (not, SWOOLE_BASE), inside php php script file "corun.php"

But the command "ps aux | grep php" on command-line is showing me the two corun.php files, i can not figure out why can that be? The whole output is as below:

fakhar   11127  0.5  0.2 511304 32660 pts/0    Sl+  16:44   0:00 php corun.php
fakhar   11128  0.1  0.0 365872 10324 pts/0    S+   16:44   0:00 php corun.php
fakhar   11131  0.0  0.0 367508 10232 pts/0    S+   16:44   0:00 php : task_worker
fakhar   11132  0.0  0.0 367508 10232 pts/0    S+   16:44   0:00 php : task_worker
fakhar   11133  0.0  0.0 367508 10232 pts/0    S+   16:44   0:00 php : task_worker
fakhar   11134  0.0  0.0 367508 10232 pts/0    S+   16:44   0:00 php : task_worker
fakhar   11135  0.6  0.1 375840 13308 pts/0    Sl+  16:44   0:00 php : worker
fakhar   11136  0.5  0.1 375840 13308 pts/0    Sl+  16:44   0:00 php : worker

I also used : sudo lsof -t -i:9501

and, got output below:
11127
11128
11131
11132
11133
11134
11135
11136

Looking forward,

Kind Regards,
Fakhar Anwar

**Comment by sy-records** [comment, marked as answer]
11127 is master process
11128 is manager process

**Reply by fakharak** [reply]
Thanks for quick response. That helps.
